.gitignore

```diff
@@ -125,3 +125,6 @@ chromedriver.exe
 ### Vagrant files ###
 .vagrant/
 gunicorn_logs.txt
+
+### Docker env files ###
+env.list
```

Comment:
What is this for? This isn't mentioned in your README and should almost certainly be removed.

Reply:
Hmm good point. I feel like I needed it for something, but perhaps you're --env-file option negates the need.